helix-rest/src/main/java/org/apache/helix/rest/server/resources/zookeeper/ZooKeeperAccessor.java

```diff
@@ -101,6 +100,26 @@ public Response get(@PathParam("path") String path, @QueryParam("command") Strin
     }
   }
 
+  @DELETE
+  @Path("{path: .+}")
+  public Response delete(@PathParam("path") String path) {
+    // Lazily initialize ZkBaseDataAccessor
+    ServerContext _serverContext =
+        (ServerContext) _application.getProperties().get(ContextPropertyKeys.SERVER_CONTEXT.name());
+    _zkBaseDataAccessor = _serverContext.getByteArrayZkBaseDataAccessor();
+
+    path = prependPath(path);
```

Comment:
Actually if we set the root path as below, we don't need to prepend the root slash, which is not that clean.
```
@Path("/zookeeper{path: /.+}")
public class ZooKeeperAccessor extends AbstractResource {
```

Reply:
Neat, changed the code accordingly. There is more code like that. But I would not change them in this PR.